client/db/bolt/db.go

```diff
@@ -848,7 +848,7 @@ func (db *BoltDB) DEXOrdersWithActiveMatches(dex string) ([]order.OrderID, error
 			}
 
 			// Inactive if refunded.
-			proofB := getCopy(mBkt, proofKey)
+			proofB := mBkt.Get(proofKey)
```

Comment:
In this method, `DEXOrdersWithActiveMatches`, the proof bytes are only used inside the db.Tx to check revoked and refunded status, but not outside the tx, so no copy is safe.